percy/snapshot.py

```diff
@@ -81,3 +81,28 @@ def percy_snapshot(driver, name, **kwargs):
     except Exception as e:
         print(f'{LABEL} Could not take DOM snapshot "{name}"')
         print(f'{LABEL} {e}')
+
+# Take screenshot on driver
+def percy_screenshot(driver, name, **kwargs):
+    if not is_percy_enabled(): return
+
+    try:
+        # Post to automateScreenshot endpoint with driver options and other info
+        response = requests.post(f'{PERCY_CLI_API}/percy/automateScreenshot', json={**kwargs, **{
```

Comment:
this endpoint name might rename. Also we need to make sure that we are able to run PoA sessions using this.

Reply:
Verified running fine